src/Components/Label/index.js

```diff
@@ -0,0 +1,16 @@
+import React, { Component } from 'react';
+import './index.css';
+
+export default class Label extends Component {
```

Comment:
I dont't think this file is supposed to be here.

Reply:
Correct - it's from another Component.